src/install-otp.ps1

```diff
@@ -7,13 +7,14 @@ Set-Location $Env:RUNNER_TEMP
 $FILE_OUTPUT="otp.exe"
 $DIR_FOR_BIN=".setup-beam/otp"
 
-Remove-Item -Recurse -Force "$DIR_FOR_BIN" -ErrorAction SilentlyContinue
 $ProgressPreference="SilentlyContinue"
 Invoke-WebRequest "https://github.com/erlang/otp/releases/download/OTP-$VSN/otp_win64_$VSN.exe" -OutFile "$FILE_OUTPUT"
 $ProgressPreference="Continue"
 New-Item "$DIR_FOR_BIN" -ItemType Directory | Out-Null
 Move-Item "$FILE_OUTPUT" "$DIR_FOR_BIN"
 Start-Process "./$DIR_FOR_BIN/$FILE_OUTPUT" /S -Wait
-Write-Output "C:/Program Files/erl-$VSN/bin" | Out-File -FilePath $Env:GITHUB_PATH -Encoding utf8 -Append
+$ErlExec = Get-ChildItem -Path "C:/Program Files/" -Recurse -Depth 2 -Filter 'erl.exe' -Name | ForEach-Object { Write-Output "C:/Program Files/$_" }
```

Comment:
Yeah, so the new approach is to find `erl.exe` and make the container folder part of the path. 🎉 , more Windows versions supported.